public/api/conf/1.0/docs/overview.md

```diff
@@ -10,6 +10,18 @@ Pull notifications remain queued for 14 days after which they are deleted from t
 
 The Pull Notifications API works in 2 discrete modes. It is recommended that your application use the Pull Notifications API as described under Retrieve pull notifications.
 
+## Retrieve pull notifications by conversation identifier
```

Comment:
this heading include conversation id but listed below are endpoints that don't have conversation id? Maybe drop `by conversation identifier`?